REST/Documentation.json

```diff
@@ -3695,6 +3755,42 @@
       ],
       "type": "object"
     },
+    "UserCreateModel": {
+      "properties": {
+        "description": {
+          "type": "string"
+        },
+        "email": {
+          "type": "string"
+        },
+        "id": {
+          "type": "integer"
+        },
+        "name": {
+          "type": "string"
+        },
+        "password": {
+          "type": "string"
+        },
+        "phoneNumber": {
+          "type": "string"
+        },
+        "surname": {
+          "type": "string"
+        },
+        "username": {
+          "type": "string"
+        },
+        "userTypeId": {
+          "type": "integer"
+        }
+      },
+      "required": [
+        "id",
```

Comment:
Isn't having a required Id a bit misleading? Id has to be 0 here so passing anything else will result in a failure.

Reply:
It's not required, it works even without id.

Reply:
Exactly, but it is marked as required.